.cursor/BUGBOT.md

```diff
@@ -0,0 +1,128 @@
+# Bugbot PR Review Guidelines
+
+## Repository-Wide Validation Framework
```

Comment:
This section is good, but I would leave as just an element hiding check for now, and we can expand it to other features as a follow-up once you see how it behaves in testing.